checker/manual/external-tools.tex

```diff
@@ -792,6 +793,24 @@ \section{tIDE\label{tide}}
 % See its documentation at \myurl{http://tide.olympe.in/}.
 \end{sloppypar}
 
+\section{NetBeans\label{netbeans}}
+To run a checker within NetBeans:
```

Comment:
Please use two blank lines above `\section{...}` and one blank line below, just because the rest of the document is written that way.